### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by nolanlawson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,19 +16,23 @@
...
+        "@lwc/compiler": "^2.30.1",
+        "@lwc/engine-dom": "^2.30.1",
+        "@lwc/engine-server": "^2.30.1",
+        "@lwc/synthetic-shadow": "^2.30.1",
+        "@lwc/wire-service": "^2.30.1",
+        "@types/jest": "^29.2.2",
+        "eslint": "^8.27.0",
+        "jest": "^29.3.1",
+        "jest-environment-jsdom": "^29.3.1",
+        "lerna": "^6.0.3",
```
Comment (nolanlawson) [inline package.json:30]: Breaking changes in Lerna don't apply to us: https://github.com/lerna/lerna/blob/main/CHANGELOG.md

File: package.json [hunk 2]
```diff
@@ -16,19 +16,23 @@
...
         "prettier": "^2.7.1",
-        "semver": "^7.3.7"
+        "semver": "^7.3.8"
+    },
+    "volta": {
+        "node": "14.21.1",
+        "yarn": "1.22.19"
```
Comment (nolanlawson) [inline package.json:36]: Figured we might as well add Volta; we were missing this. Circle CI is testing Node 14.

File: packages/@lwc/jest-transformer/src/transforms/__tests__/utils/test-transform.js [hunk 3]
```diff
@@ -36,6 +36,11 @@ function transform(plugin, opts = {}) {
     };
 }
 
+function collapseNewlines(str) {
+    // At some point Babel stopped emitting duplicate newlines
+    return str.replace(/\n{2,}/g, '\n');
+}
+
```
Comment (divmain) [inline packages/@lwc/jest-transformer/src/transforms/__tests__/utils/test-transform.js:43]: This is just to avoid having to update fixtures, yes?

**Review by divmain — APPROVED** [review]

File: packages/@lwc/jest-transformer/src/transforms/__tests__/utils/test-transform.js (hunk 3 above)
Comment (nolanlawson) [reply packages/@lwc/jest-transformer/src/transforms/__tests__/utils/test-transform.js:43]: Correct.

**Pull request merged**